packages/frontend/src/utils/format.ts

```diff
@@ -13,6 +13,9 @@ export function formatPercentage(amount?: number | null): string {
 
 export function formatTokenAmount(amount?: number | null): string {
     if (amount && amount < 0.0001) return "<0.0001";
+    if (amount && amount < 1) {
+        return `${numeral(amount).format("0,0.0000")}`;
```

Comment:
I think we can have 2 min decimals and 2 optional, so `0,0.00[00]`, would that be ok?

Reply:
How about this?

```
export function formatTokenAmount(amount?: number | null): string {
    if (amount && amount < 0.0001) return "<0.0001";
    return `${numeral(amount).format("0,0.0[000]a")}`;
}

```

Reply:
Could work, but the `a` should be optional, there's a change on develop for this that's causing the conflicts.

Reply:
Merged✅